src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -7,7 +7,9 @@ import {atom, useRecoilState} from 'recoil';
 
 import AnonymousMessageRepo from '../../../service/repo/anonymousMessageRepo';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
-import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
+import UseChatUtilsHook, {
+  ContactScreenPayload
+} from '../../../../types/hooks/screens/useChatUtilsHook.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `UseChatUtilsHook` and `ContactScreenPayload` has been updated. Ensure that the `ContactScreenPayload` type is correctly defined in the `useChatUtilsHook.types` file.
```diff
- import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
+ import UseChatUtilsHook, {
+   ContactScreenPayload
+ } from '../../../../types/hooks/screens/useChatUtilsHook.types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->